app/views/layouts/layout.slim

```diff
@@ -173,127 +235,144 @@ html lang="en"
-                        | &nbsp;
-                        span.caret
-                        span.sr-only Toggle Dropdown
-                      ul.dropdown-menu.notebooksmenu id='dropdownMenu'
+                              a href="#{reviews_path}"
+                                span.tab All Reviews
+                            -if @user.admin? or user_signed_in?
+                              li.divider role="separator"
+                              li
+                                span
+                                  ' Logged in as:
+                                  br
+                                  strong ==@user.user_name
+                              li.divider role="separator"
+                            -if @user.admin?
+                              li
+                                a href="/admin"
+                                  span.tab Admin
+                            -if user_signed_in?
+                              li
+                                == link_to("Logout", destroy_user_session_path, :method => :delete)
+        hr.hidden
         -if request.path != "/"
-          div id='expandableSearchDropdown'
+          div id="expandableSearchDropdown" style="display: none"
             div.expandable-search-inner
               div.expandable-search-input
-                form role='search' action='/notebooks'
+                form role="search" action="/notebooks"
                   button.btn.search-submit
                     span.glyphicon.glyphicon-search
-                      span.visually-hidden Search
-                  input.searchFieldBox.form-control type='text' placeholder='Search' name='q' value="#{params[:q] || ''}"
-                  input type='hidden' name='sort' value='score'
+                      span.sr-only Search
+                  input.searchFieldBox.form-control type="text" placeholder="Search" name="q" value="#{params[:q] || ''}"
+                  input type="hidden" name="sort" value="score"
         -if @warning != nil
-          div class = "alert center #{@warning.level == 'warning' ? 'alert-danger' : 'alert-info'}" id='galleryWarning' role="banner"
+          div class = "alert center #{@warning.level == "warning" ? "alert-danger" : "alert-info"}" id="galleryWarning" role="banner"
             strong #{@warning.message}
         div.modals
-          -url_check = request.path.split('/')
-          ==render partial: 'modals/login_modal'
-          ==render partial: 'modals/upload.slim'
-          ==render partial: 'modals/environment_loading'
-          ==render partial: 'modals/new_group.slim'
+          -url_check = request.path.split("/")
+          ==render partial: "modals/login_modal"
+          ==render partial: "modals/upload.slim"
+          ==render partial: "modals/environment_loading"
+          ==render partial: "modals/new_group.slim"
           -if url_check[1] == "groups" && url_check[2] != nil && url_check[2][0..3] =~ /\d/
```

Comment:
Need to check if the group object exists before rendering this.  There are somtimes errors getting thrown here for null objects not having an ID